Ted Talk/Ted Talk/Model/TedTalkData.swift

```diff
@@ -6,22 +6,59 @@
 //
 
 import Foundation
+import RealmSwift
 
-struct TedTalkData: Codable {
-    var comments: Int
-    var description: String
-    var duration: Int
-    var event: String
-    var film_date: Int
-    var languages: Int
-    var main_speaker: String
-    var name: String
-    var num_speaker: Int
-    var published_date: Int
-    var speaker_occupation: String
-    var tags: [String]
-    var title: String
-    var url: String
-    var views: Int
+class TedTalkData: Object, Codable {
+    @Persisted var comments: Int
+    @Persisted var descript: String
+    @Persisted var duration: Int
+    @Persisted var event: String
+    @Persisted var film_date: Int
+    @Persisted var languages: Int
+    @Persisted var main_speaker: String
+    @Persisted var name: String
+    @Persisted var num_speaker: Int
```

Comment:
```suggestion
    @Persisted var numberSpeakers: Int
```

i'd use this naming.. i think we're not using snake coding in our apps.. but.. if you are planning to have a domain layer.. i'd keep it as it's and then change it there..